ReleaseHistory.md

```diff
@@ -16,9 +16,10 @@
 - NEW => new feature 
 
 ## UNRELEASED
+* DEP: Remove `Microsoft.CodeAnalysis`. [#934](https://github.com/microsoft/binskim/pull/934)
+* DEP: Remove `Microsoft.CodeAnalysis.NetAnalyzers`. [#934](https://github.com/microsoft/binskim/pull/934)
```

Comment:
>Microsoft.CodeAnalysis.NetAnalyzers [](http://example.com/codeflow?start=15&length=35)

this isn't a product change, strictly speaking, yes?

the CodeAnalysis binary removal is, though.